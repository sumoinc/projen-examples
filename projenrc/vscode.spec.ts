import { TypeScriptProject } from "projen/lib/typescript";
import { synthSnapshot } from "projen/lib/util/synth";

describe("Success Conditions", () => {
  it("snapshot should match", () => {

    const project = new TypeScriptProject({
      name: "foo",
      defaultReleaseBranch: "main",
    });

    const content = synthSnapshot(project);
    expect(content[".vscode/settings.json"]).toMatchSnapshot();
  });
});
