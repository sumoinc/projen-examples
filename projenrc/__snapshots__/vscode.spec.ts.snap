// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Success Conditions snapshot should match 1`] = `undefined`;
